MiniApp/Classes/Protocols/Protocol Extensions/MiniAppUserInfoProtocolExtension.swift

```diff
@@ -0,0 +1,9 @@
+public extension MiniAppUserInfoDelegate {
```

Comment:
I wonder if it makes more sense to define the extension in the same file as the protocol? Especially because it's a pretty short protocol right now. This would keep them closer together so it's obvious that there's a default implementation of the protocol.

Reply:
I was just thinking the same thing

Reply:
Done